### PR TITLE
Hide the red dot from payments button

### PR DESCRIPTION
**Pull request by kidinov**
Closes: #7022

### Description

The PR hides the red badge after a user clicks on it

### Testing instructions

* Go to the More screen
* Notice red badge on the payments icon
* Click on the icon and come back
* Notice that the badge is gone

### Images/gif

https://user-images.githubusercontent.com/4923871/181201342-692a7d5d-a57e-4c31-ab3b-6c2a6e87643a.mp4

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/test/kotlin/com/woocommerce/android/ui/moremenu/MoreMenuViewModelTests.kt [hunk 1]
```diff
@@ -80,4 +82,36 @@ class MoreMenuViewModelTests : BaseUnitTest() {
         // THEN
         verify(moreMenuNewFeatureHandler).markNewFeatureAsSeen()
     }
+
+    @Test
+    fun `given user never clicked payments, when building state, then badge displayed`() = testBlocking {
```
Comment (AnirudhBhat) [inline WooCommerce/src/test/kotlin/com/woocommerce/android/ui/moremenu/MoreMenuViewModelTests.kt:89]: ❓ It looks like this and the below test are the same or am I missing something?

File: WooCommerce/src/main/kotlin/com/woocommerce/android/AppPrefs.kt [hunk 2]
```diff
@@ -702,6 +705,13 @@ object AppPrefs {
         setBoolean(UndeletablePrefKey.USER_SEEN_NEW_FEATURE_MORE_SCREEN, true)
     }
 
+    fun isPaymentsIconWasClickedOnMoreScreen(): Boolean =
```
Comment (AnirudhBhat) [inline WooCommerce/src/main/kotlin/com/woocommerce/android/AppPrefs.kt:809]: ❓ What do you think about writing tests for this change

**Comment by AnirudhBhat** [comment]
LGTM @kidinov Just left a minor comment which is not blocking and a question about the test being similar.

**Review by AnirudhBhat — APPROVED** [review]

File: WooCommerce/src/test/kotlin/com/woocommerce/android/ui/moremenu/MoreMenuViewModelTests.kt (hunk 1 above)
Comment (kidinov) [reply WooCommerce/src/test/kotlin/com/woocommerce/android/ui/moremenu/MoreMenuViewModelTests.kt:89]: Good catch. Copilote made a mistake =)

Fixed - 95e37757ce12b25a3cedc88169054b03117b125e

File: WooCommerce/src/main/kotlin/com/woocommerce/android/AppPrefs.kt (hunk 2 above)
Comment (kidinov) [reply WooCommerce/src/main/kotlin/com/woocommerce/android/AppPrefs.kt:809]: Yep! 3e5b06bfd54565985c56540085422b0f763a9f59

**Pull request merged**